source/frontend/src/features/mapSideBar/acquisition/common/GenerateForm/hooks/useGenerateH0443.test.tsx

```diff
@@ -102,6 +106,122 @@ describe('useGenerateH0443 functions', () => {
     });
   });
 
+  it('makes requests to Organization when Property Coordinator is an organization with a Primary Contact', async () => {
+    const organizationPersonMock = getMockPerson({ id: 3, firstName: 'JONH', surname: 'Doe' });
+    getPersonConceptFn.mockResolvedValue(Promise.resolve({ data: organizationPersonMock }));
+
+    const organizationMock = getMockOrganization();
+    getOrganizationConceptFn.mockResolvedValue(Promise.resolve({ data: organizationMock }));
```

Comment:
nit: no need to use `Promise.resolve(...)` when calling **mockResolvedValue** - jest already returns a promise. 
Please change to 
```tsx
  getPersonConceptFn.mockResolvedValue({ data: organizationPersonMock });
```

Reply:
I get an error with that suggestion. 
![image](https://github.com/bcgov/PSP/assets/4305638/34129c92-f04f-448d-9b45-092bee3d4ca4)